internal/issuer/clients/cert/requests_test.go

```diff
@@ -0,0 +1,155 @@
+package cert
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var exampleBytes = []byte("example")
+
+const (
+	PostCert         = "PostCert"
+	GetCert          = "GetCert"
+	FailGetCert      = "FailGetCert"
+	InvalidResponse  = "InvalidResponse"
+	TestURL          = "https://test.com/"
+	DownloadEndpoint = "download/"
+	GUID             = "12345678/"
+)
+
+func TestRequests(t *testing.T) {
```

Comment:
Each function should be tested separately, so there should be a `TestPostCertificate` and `TestDownloadCertificate`